ta/os_test/os_test.c

```diff
@@ -329,7 +323,7 @@ static TEE_Result test_malloc(void)
 static TEE_Result test_properties(void)
 {
 	TEE_Result res = TEE_ERROR_GENERIC;
-	TEE_PropSetHandle h;
+	TEE_PropSetHandle h = { 0 };
```

Comment:
`{ }`

Reply:
Right, so `TEE_HANDLE_NULL` should be used instead.

Reply:
ok